test/index.test.ts

```diff
@@ -1,18 +1,18 @@
-import * as u8a from 'uint8arrays'
-import dagJose, { createDagJWS } from '../src/index'
+/* eslint-env jest */
+
+import * as dagJose from '../src/index'
+import { createDagJWS } from './signing.test'
 import sFixtures from './__fixtures__/signing.fixtures'
 import eFixtures from './__fixtures__/encryption.fixtures'
-// TODO - multiformats imports not working, ignoring for now.
-//import multiformats from 'multiformats'
-//import legacy from 'multiformats/legacy.js'
+import { convert as toLegacyIpld } from 'blockcodec-to-ipld-format'
 import IPLD from 'ipld'
 import ipldInMem from 'ipld-in-memory'
-import CID from 'cids'
-import { EllipticSigner } from 'did-jwt'
+import { CID } from 'multiformats/cid'
+import { ES256KSigner } from 'did-jwt'
 
 describe('dag-jose codec', () => {
   describe('DagJWS', () => {
-    it('Encode compact jws', () => {
+    it.only('Encode compact jws', () => {
```

Comment:
Should not be `.only`?

Reply:
whoops! leftover from debugging, will remove